### PR TITLE
fix: remove output warnings by changing variables to target-org and disable-telemetry

### PR DESCRIPTION
**Pull request by daphne-sfdc**
### What issues does this PR fix or reference?
[@W-13775011@] (https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001WMM9DYAX/view)

### Functionality Before
When you run "SFDX: Set a Default Org" in VSCode, the output table in the Outputs tab displays "defaultusername" in the "Name" column, which is the original value that we want to change.

### Functionality After
(1) When you run "SFDX: Set a Default Org" in VSCode, the output table in the Outputs tab displays "target-org" in the "Name" column, which matches the new value of the variable.
(2) This PR contains a refactor that takes the values of the TARGET_ORG_KEY and SF_CONFIG_DISABLE_TELEMETRY constants from constants.ts instead of from node_modules.

### CODE REVIEW

File: packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts [hunk 1]
```diff
@@ -8,11 +8,11 @@
 import { GlobalCliEnvironment } from '../cli';
 import { ConfigUtil } from '../config/configUtil';
 
-export const ENV_SFDX_DISABLE_TELEMETRY = 'SFDX_DISABLE_TELEMETRY';
+export const ENV_SF_DISABLE_TELEMETRY = 'SFDX_DISABLE_TELEMETRY';
```
Comment (CristiCanizales) [inline packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts:11]:
```suggestion
export const ENV_SF_DISABLE_TELEMETRY = 'SF_DISABLE_TELEMETRY';
```

File: packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {
   StateAggregator
 } from '@salesforce/core';
 import { ConfigUtil, workspaceUtils } from '../../../src';
```
Comment (CristiCanizales) [inline packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts:7]:
```suggestion
import { ConfigUtil, TARGET_ORG_KEY, workspaceUtils } from '../../../src';
```

File: packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import {
...
+import { TARGET_ORG_KEY } from '../../../src/constants';
```
Comment (CristiCanizales) [inline packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts:8]:
```suggestion
```

File: packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts [hunk 4]
```diff
@@ -5,6 +5,7 @@ import {
   StateAggregator
```
Comment (CristiCanizales) [inline packages/salesforcedx-utils-vscode/test/jest/config/configUtil.test.ts:5]: We should delete line 4 too as OrgConfigProperties is not used anymore

File: packages/salesforcedx-utils-vscode/test/unit/config/configUtil.test.ts
Comment (CristiCanizales) [inline packages/salesforcedx-utils-vscode/test/unit/config/configUtil.test.ts:1]: ⬆️ Same comments for this file

**Review by CristiCanizales — CHANGES_REQUESTED** [review]

File: packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts (hunk 1 above)
Comment (daphne-sfdc) [reply packages/salesforcedx-utils-vscode/src/telemetry/cliConfiguration.ts:11]: Oops missed that one yesterday!

**Review by CristiCanizales — APPROVED** [review]
LGTM 🙂 ✅ 

🚢

**Pull request merged**